index.html

```diff
@@ -109,7 +109,6 @@ <h1 id = "page-title" class = "drop-shadow-2xl text text-center font-Rowdies tex
     
         <!-- Current Results -->
         <section>
-            <h2>SYNOPSIS</h2>
```

Comment:
@itsbbea I don't think we should remove line 112

I think we should use Kayden's suggestion of adding some id to the h2 tag and setting the id to have styling display:none rather than removing the line. We can then set the attribute later to display:block

Reply:
That's fine, I agree. I removed it because I was trying to edit this on my side and thought it would be best to remove the synopsis title, but styling it with display: none would resolve this issue.